core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -707,7 +707,7 @@ class LogManager(logDirs: Seq[File],
    * @param topicPartition The partition whose log needs to be returned or created
    * @param config The configuration of the log that should be applied for log creation
    * @param isNew Whether the replica should have existed on the broker or not
-   * @param isFuture True iff the future log of the specified partition should be returned or created
+   * @param isFuture True if the future log of the specified partition should be returned or created
```

Comment:
The `iff` was probably intentional, if a bit pedantic. I'm fine with the change though.

Reply:
Oh.. I did not know that `iff` was actually valid. I thought that it was a typo :)

Reply:
Yeah, it's a mathematical short-hand for "if and only if." But the "only if" part is implicit here.